app/src/components/roles/AddRoleCategoryForm.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+  <div>
+    <h1 class="font-bold text-2xl">{{ isNew ? 'Create' : 'Update' }} Role Category</h1>
+    <hr class="mt-4" />
+  </div>
+
+  <div class="max-h-[70vh] overflow-auto">
+    <!--General Inputs-->
+    <section class="flex flex-col gap-2">
+      <label class="input input-bordered flex items-center gap-2 input-md mt-4">
+        <span class="w-24">Name</span>
+        <input
+          type="text"
+          class="grow"
+          placeholder="Enter a role category name"
+          v-model="roleCategory.name"
+        />
+      </label>
+      <label class="input input-bordered flex items-center gap-2 input-md">
+        <span class="w-24">Description</span>
+        <input
+          type="text"
+          class="grow"
+          placeholder="Enter a short description"
+          v-model="roleCategory.description"
+        />
+      </label>
+    </section>
+
+    <div v-if="props.isNew">
+      <!--Roles-->
+      <section class="mt-10">
+        <div
+          v-for="(input, index) in roleCategory.roles"
+          :key="index"
+          class="collapse collapse-plus bg-base-200 border mt-2"
+        >
+          <input type="radio" name="my-accordion-3" />
```

Comment:
The accordions do not have the right behavior. (Does not toggle)
Use the AccordionComponent.vue component instead.

Reply:
Okay, I'll effect on a new request for the front-end